bin/kubectl-crossplane-stack-install

```diff
@@ -75,7 +75,8 @@ STACK_IMAGE_NAME="${1}"
 # For kubernetes fields, we aren't able to use slashes, and
 # slashes are common for docker image names. So we remove the
 # slashes before we use the name for kubernetes resource fields.
-KUBEY_STACK_IMAGE_NAME=$( echo "${STACK_IMAGE_NAME}" | tr '/' '-' )
+# We also can't use colons and don't want to include tags in the name.
+KUBEY_STACK_IMAGE_NAME=$( echo "${STACK_IMAGE_NAME}" | tr '/' '-' | sed 's/:.*//')
```

Comment:
Looks like it doesn't quite work yet. With `set -x` enabled:

```bash
$ ./bin/kubectl-crossplane-stack-install crossplane/stack-gcp:master
+ CLUSTER_STACK=
+ CLUSTER_OPT=
+ NAMESPACE_OPT=
+ POSITIONAL=()
+ [[ 1 -gt 0 ]]
+ opt=crossplane/stack-gcp:master
+ case $opt in
+ POSITIONAL+=("$1")
+ shift
+ [[ 0 -gt 0 ]]
+ '[' 1 -eq 0 ']'
+ set -- crossplane/stack-gcp:master
+ STACK_IMAGE_NAME=crossplane/stack-gcp:master
++ echo crossplane/stack-gcp:master
++ tr / -
++ sed 's/:.*//'
+ KUBEY_STACK_IMAGE_NAME=crossplane-stack-gcp
+ STACK_NAME=crossplane-stack-gcp
+ kubectl crossplane stack generate-install crossplane/stack-gcp:master
+ kubectl apply -f -
The StackInstall "crossplane-stack-gcp:master" is invalid: metadata.name: Invalid value: "crossplane-stack-gcp:master": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')
```

Looks like we would need to put the same logic into `generate_install`. Or, even better, we could delete `generate_install` now that `install` supports namespace : )

Reply:
`generate_install` is still useful for running with `| kubectl --context=... apply -f -`

For whatever reason, `kubectl --context=... crossplane` isn't recognized.

Reply:
I wasn't able to reproduce the error you received. I tried on Linux and OS/X.

I've pushed an update for `generate-install` and added a space before a closing parenthesis.

Would you mind trying again with the latest changes and replying with the result of the following if it still isn't working:

```
# should be repo-foo
echo $( echo "repo/foo:bar" | tr '/' '-' | sed 's/:.*//' )
```